app/src/main/java/com/veephealthoutloud/healthoutloud/Classes/Requests.java

```diff
@@ -0,0 +1,102 @@
+package com.veephealthoutloud.healthoutloud.Classes;
+
+import android.app.Application;
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonArrayRequest;
+
+import com.veephealthoutloud.healthoutloud.ApplicationActivity;
+import com.veephealthoutloud.healthoutloud.Classes.Post;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.lang.reflect.Array;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+
+import java.util.Date;
+
+/**
+ * Used to send requests to backend api
+ */
+
+public class Requests {
+
+    private static String endpoint = "http://healthoutloud-api.herokuapp.com";
+
+    private ArrayList<Post> processJSONPosts(JSONArray posts)
+    {
+
+        ArrayList<Post> allPosts = new ArrayList<>();
+        Post _post;
+
+        for(int index = 0; index < posts.length(); index++) {
+
+            try {
+                JSONObject JSONPost = (JSONObject) posts.get(index);
+                JSONArray JSONFeelings = JSONPost.getJSONArray("feeling");
+
+                ArrayList<String> postFeelings = new ArrayList<>();
+                for(int feelingsIndex = 0; feelingsIndex < JSONFeelings.length(); feelingsIndex++){
+                    postFeelings.add((String) JSONFeelings.get(feelingsIndex));
+                }
+
+                String dateStr = JSONPost.getString("date");
+                SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+                Date postDate = sdf.parse(dateStr);
+
+                _post = new Post(JSONPost.getString("_id"), JSONPost.getString("postBody"),
+                        postDate, postFeelings);
+
+                allPosts.add(_post);
+            }
+            catch (JSONException e) {
+                // TODO: Exception Handler
+            } catch (ParseException p) {
+                // TODO: Exception Handler
+            }
+        }
+
+        return allPosts;
+
+    }
+
+    public void getAllPosts()
+    {
+
+        final String postsURL = endpoint + "\\posts";
```

Comment:
This looks like a typo but not sure since i'm not familiar w/ volley (the double \\)